src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -155,7 +155,7 @@ static void queryForward(ConnectionInfo* ciP, QueryContextRequest* qcrP, Format
   if ((out == "error") || (out == ""))
   {
     qcrsP->errorCode.fill(SccContextElementNotFound, "");
-    LM_E(("Runtime Error (error forwarding 'Query' to providing application)"));
+    LM_W(("Runtime Error (error forwarding 'Query' to providing application)"));
```

Comment:
This is not actually part of the topic of the PR, but I have seem while fixing other things.

This condition shouldn't be loged as ERROR, as it is not a problem in the CB itself, but in a external system. Thus, I think that the righ log level is WARNING.